README.md

```diff
@@ -17,12 +20,20 @@ What's wrong with this mongoose code and how might we fix it?
 (Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
 
 ```js
+
+
 var results = AuthorModel.find({name: "Bob"});
 console.log(results);
 ```
 
 ```js
-// Your answer...
+The find method needs call beck function to know what to do with the result it finds.
+
+var results = AuthorModel.find({name: "Bob"},function(result){
+console.log(results);
+});
+
```

Comment:
👍 I'll give it to you, but that doesn't explain why `results` isn't an instance of an Author.